ui/pages/confirmations/components/confirm/footer/footer.tsx

```diff
@@ -39,6 +121,18 @@ const Footer = () => {
     return false;
   });
 
+  const [alertModalVisible, setAlertModalVisible] = useState<boolean>(false);
+  const alertOwnerId = currentConfirmation?.id as string;
+  const { alerts, isAlertConfirmed } = useAlerts(alertOwnerId);
+  const unconfirmedAlerts = alerts.filter(
+    (alert) => alert.field && !isAlertConfirmed(alert.key),
+  );
+  const hasUnconfirmedAlerts = unconfirmedAlerts.length > 0;
```

Comment:
Even if all the alerts have been acknowledge via the inline alerts, I understood we still want the modal to be displayed as a final confirmation.

My understanding of this current implementation is the user gets the alerts to navigate through, exactly the same as the inline alert.

But I believe the Figma requests an overview modal when clicking confirm, which includes that `Review alerts` link and a final confirmation checkbox?

Perhaps easier to encapsulate that in a separate component such as `ConfirmAlertModal` that uses an `AlertModal` internally and then the link loads the `MultipleAlertModal`?

Reply:
Do you mean the friction modal? My bad I thought this would be done in the last ticket.
I will work on it.
![image](https://github.com/MetaMask/metamask-extension/assets/45455812/3e243674-6457-46a0-9877-56fdd318e275)

Reply:
Added friction modal [here](https://github.com/MetaMask/metamask-extension/pull/24049/commits/c44abe34303f83d82881a7b0b5115edc51b15afc)